### PR TITLE
Us133 borrower confirms deed

### PR DESCRIPTION
**Pull request by tsangsimon**

### CODE REVIEW

**Comment by Kirosoft** [comment]
:+1:

File: application/deed/views.py [hunk 1]
```diff
@@ -106,3 +108,51 @@ def delete_borrower(borrower_id):
...
     else:
         return jsonify({"id": borrower_id}), status.HTTP_200_OK
+
+
+@deed_bp.route('/<deed_reference>/sign', methods=['POST'])
+def sign_deed(deed_reference):
+    request_json = request.get_json()
+    borrower_token = request_json['borrower_token']
+    result = Deed.query.filter_by(token=str(deed_reference)).first()
+
```
Comment (Kirosoft) [inline application/deed/views.py:117]: This should be protected with the header id like the main get endpoint. Calling the get method on the model will do the header parsing for you.

File: application/deed/views.py [hunk 2]
```diff
@@ -106,3 +108,51 @@ def delete_borrower(borrower_id):
...
+    return jsonify({"deed": result.deed}), status.HTTP_200_OK
+
+
+@deed_bp.route('/<deed_reference>/make-effective', methods=['POST'])
+def make_effective(deed_reference):
+    return status.HTTP_200_OK
+
+
+def update_deed_signature_timestamp(deed, borrower_token):
```
Comment (Kirosoft) [inline application/deed/views.py]: push this function into the service.py file to keep the top level routes file as clean as possible pls

Comment (tsangsimon) [inline application/deed/views.py]: yep like this

**Comment by Kirosoft** [comment]
Unit tests to be added as part of US133f.
Good work :-)
:+1:

**Pull request merged**